variables.tf

```diff
@@ -46,6 +46,9 @@ variable "apex_domain" {
 variable "domain_name" {
   type = string
 }
+variable "msi_name" {
```

Comment:
Can we update the  Readme to cover off what AAD pod identity  is about and the variables it introduces?